src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  & :last-child {
+    margin-right: 0;
+  }
+
+  &__star {
+    background-image: $star;
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+
+    @include square($star-size);
+  }
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: $star-active;
+}
```

Comment:
Paul probably wants this in a loop

Reply:
Yep, Paul wants it to be writtend with sass loop.